_posts/2019-12-18-sentry-the-guardian-of-your-quarkus-app.adoc

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: 'Sentry Error Tracker - The new guardian of your Quarkus application'
+date: 2019-12-18
```

Comment:
You should probably adjust the date once you are done with the adjustments (header + file name).